geo_data_br/points.py

```diff
@@ -5,14 +5,15 @@
 from appdirs import user_cache_dir
 import os.path
 
-def data_on_points(points: List[Point], level='best'):
+def data_on_points(points: List[Point], column_name="coordinates", level='best'):
     '''points is a list of longitute, latitude tuples or shapely.geometry.Point objects'''
     if level not in ('best', 'all'):
         raise TypeError('choose a suported_level')
-    if not points:
+    if len(points) == 0:
```

Comment:
pq isso?

Reply:
acho q com o ==0 eh mais claro! Acho q o mais importante aqui eh botar um commentario `# len used so that comparisson works with lists, series and dataframes`